blog_tags.json

```diff
@@ -237,7 +239,7 @@
         },
         {
             "name": "developer experience",
-            "posts": ["liberty-bikes",
+            "posts": ["java17-210010", "liberty-bikes",
```

Comment:
on second thought, can you create a new tag called "Java SE" and add this post to it, along with any previously released posts that talk about Java SE.